lib/nudge_web/controllers/event_controller.ex

```diff
@@ -0,0 +1,27 @@
+defmodule NudgeWeb.EventController do
+  use NudgeWeb, :controller
+  alias Nudge.Sites
+  alias Nudge.Sites.Event
+
+  def new(conn, _params) do
+    changeset = Sites.change_event(%Event{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"event" => event_params}) do
```

Comment:
We need a service module for events.
Events will have more things to implement, and those things can't be part of controller. Let's extract these code to a svc module and use context inside it